README.md

```diff
@@ -3,6 +3,7 @@
 # SPIFFE Helm Charts
 
 [![Apache 2.0 License](https://img.shields.io/github/license/spiffe/helm-charts?style=for-the-badge)](https://opensource.org/licenses/Apache-2.0)
+[![Development Phase](https://github.com/spiffe/spiffe/blob/main/.img/maturity/dev.svg)](https://github.com/spiffe/spiffe/blob/main/MATURITY.md#development)
```

Comment:
I think this needs to be in README.md.gotmpl

Reply:
I dont think we have one for the main readme

Reply:
https://github.com/spiffe/helm-charts/blob/main/charts/spire/README.md.gotmpl ?

Reply:
That's in a sub chart. We want this to be in the root of the repo.

Reply:
Oh. Chart vs root readme. Your right. sorry.